README.md

```diff
@@ -13,3 +32,20 @@ git clone https://github.com/mozilla-releng/maven-lambda
 cd maven-lambda
 pip install maven-lambda
 ```
+
+## Links
+
+ * Production instance: https://maven.mozilla.org/
+ * Staging instance: https://maven-default.stage.mozaws.net/
+
+## Deployment
+
+The process is quite manual. See [bug 1589065 comment 27](https://bugzilla.mozilla.org/show_bug.cgi?id=1589065#c27) for instance. The idea is:
+
+ 1. let Taskcluster build a push to the master branch
+ 2. Get the direct link to `function.zip`
+ 3. Needinfo CloudOps on Bugzilla with that direct link.
+
+## Debugging
+
+Contact the CloudOps team to get access to the logs. They may refer you to tools like [awslogs](https://github.com/jorgebastida/awslogs).
```

Comment:
Note to self: This reminds me it's just the two of us with access there. I should escalate this to CloudOps at some point.